build.gradle

```
@@ -199,20 +219,31 @@ def latestJava = 17
 def oldestJava = 8
 
 tasks.withType(JavaCompile).configureEach {
-	options.compilerArgs += ["-source", "" + oldestJava, "-target", "" + oldestJava]
+	options.compilerArgs += ['-source', '' + oldestJava, '-target', '' + oldestJava]
+}
+
+createTestTask('JUnitQuick', 'Runs JUnit tests on one environment being the latest supported Java and Minecraft.', 'src/test/skript/environments/' + latestEnv, false, latestJava, false, true)
+createTestTask('JUnitJava17', 'Runs JUnit tests on all Java 17 environments.', 'src/test/skript/environments/java17', false, latestJava, false, true)
```

If you could store all the paths (not just these ones) in variables it would look much better
```suggestion
createTestTask('JUnitJava17', 'Runs JUnit tests on all Java 17 environments.', ENVIRONMENTS_PATH + '/java17', false, latestJava, false, true)
```

---

#5299 already makes these changes